src/lib389/lib389/cli_conf/plugins/rootdn_ac.py

```diff
@@ -20,8 +22,71 @@
 }
 
 
+def validate_args(args):
+    # validate the args
+    if args.close_time is not None:
+        try:
+            int(args.close_time)
+        except:
+            raise ValueError("The close time must be a 4 digit number: HHMM")
+        if len(args.close_time) != 4:
+            raise ValueError("The close time must be a 4 digit number: HHMM")
+        hour = int(args.close_time[:2])
+        if hour < 0 or hour > 23:
+            raise ValueError(f"The hour portion of the time is invalid: {hour}  Must be between 0 and 23")
+        min = int(args.close_time[-2:])
+        if min < 1 or min > 59:
+            raise ValueError(f"The minute portion of the time is invalid: {min}  Must be between 1 and 59")
+
+    if args.open_time is not None:
+        try:
+            int(args.open_time)
+        except:
+            raise ValueError("The open time must be a 4 digit number: HHMM")
+        if len(args.open_time) != 4:
+            raise ValueError("The open time must be a 4 digit number: HHMM")
+        hour = int(args.open_time[:2])
+        if hour < 0 or hour > 23:
+            raise ValueError(f"The hour portion of the time is invalid: {hour}  Must be between 0 and 23")
+        min = int(args.open_time[-2:])
+        if min < 1 or min > 59:
+            raise ValueError(f"The minute portion of the time is invalid: {min}  Must be between 1 and 59")
```

Comment:
Maybe I miss something but I think it should allow 0 to 59:

```
$ dsconf localhost plugin root-dn set --close-time 0100
Error: The minute portion of the time is invalid: 0  Must be between 1 and 59
```

Reply:
Heh, you're right :-)  I'll get that changed and merged...